src/asm_writing/rewriter.cpp

```diff
@@ -1148,6 +1146,20 @@ void Rewriter::_setupCall(bool has_side_effects, llvm::ArrayRef<RewriterVar*> ar
 #endif
 }
 
+void Rewriter::_callOptimalEncoding(assembler::Register tmp_reg, void* func_addr) {
```

Comment:
Oh sorry you handled that :) I keep forgetting that we pick the destination before we start writing everything out.